examples/android/android-studio/Cukeulator/README.md

```diff
@@ -0,0 +1,60 @@
+## Cukeulator Example Test
+This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
+
+### Setup
+Features must be placed in `assets/features/`. Subdirectories are allowed.
+
+The `app/libs` should contain `cucumber-android-*.jar`. Please build it following [cucumber-jvm/android]
+(https://github.com/cucumber/cucumber-jvm/tree/master/android) instructions and copy to `app/libs` folder.
+
+The rest of the dependencies are added automatically in `app/build.gradle`.
+
+
+### Using gradle
+
+To build the test apk:
```

Comment:
in case all the following steps are just about running the tests, why not simply say to run ```./gradlew connectedCheck```?